src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1,8 +1,8 @@
 # --------------------------------------------------------------------------------------------
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for license information.
+# Copyright (c) Microsoft Corporation.  All rights reserved.
+# Licensed under the MIT License.  See License.txt in the project root for
+# license information.
```

Comment:
Not sure why `azdev` isn't calling this out this filename appropriately, but other PRs are passing, and you are editting a header here. I'd try removing the changes to these lines and see if that unblocks you. Regardless, @tjprescott and I will follow up here.

Reply:
https://github.com/Azure/azure-cli-dev-tools/issues/98 opened up to track this.